### ISSUE TITLE
icons are invisible in light mode on windows with high contrast mode activated

### ISSUE THREAD

**Issue by o-hook** [porsche-design-system/porsche-design-system#1661]
Description  
---
Icons in PDS dark mode are currently not visible on windows if high contrast mode Is active (see images).

To Reproduce
---
On windows...
1. Go to https://designsystem.porsche.com/v2/components/icon/examples 
2. Activate high contrast mode
3. See error

Expected behavior
---
Icons should also be visible in high contrast mode in both themes (light/dark)

Screenshots
---
![Inkedhigh-contrast-dark](https://user-images.githubusercontent.com/40788023/141259725-fb75f571-3bd4-48da-badb-a6de36c841b5.jpg)
![Inkedhigh-contrast-light](https://user-images.githubusercontent.com/40788023/141259730-df089c70-4e06-4e11-9858-b94d9c1dd66c.jpg)

More Information
---
https://github.com/w3c/csswg-drafts/issues/6310 
Solution: https://melanie-richards.com/blog/currentcolor-svg-hcm/

Test Environment
---
Platform/Browser/Version: Windows, high contrast mode
